### PR TITLE
Fix few bbe descriptions and formatting

### PR DESCRIPTION
**Pull request by lochana-chathura**
## Purpose
$subject.

### CODE REVIEW

File: examples/transaction-statement/transaction_statement.description [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // Ballerina provides support for interacting with a transaction manager.
-// Compile-time guarantees that transactions are bracketed with begin and commit or rollback.
-// The region in the middle, is typed as being a transactional context.
-// Ballerina does not have a transactional memory and includes transaction manager.
-// Current transaction is a part of the execution context of a strand.
+// Compile-time guarantees that transactions are bracketed with `begin` and `commit` or `rollback`.
```
Comment (lochana-chathura) [inline examples/transaction-statement/transaction_statement.description:2]: Not sure about `begin` @pcnfernando @gimantha can you verify?

**Review by praneesha — APPROVED** [review]

File: examples/check-semantics/check_semantics.bal [hunk 2]
```diff
@@ -3,7 +3,7 @@ import ballerina/io;
 public function main() returns error? {
     do {
         // If either `foo()` or `bar()` invocations returns an `error`,
-        // the error will be returned from the `main` function and execution
+        // the `error` will be returned from the `main` function and execution
```
Comment (MaryamZi) [inline examples/check-semantics/check_semantics.bal:6]: We emphasize keywords or identifiers. So if we are emphasizing error here we'll have to say something like
```suggestion
        // the `error` value will be returned from the `main` function and execution
```

File: examples/check-semantics/check_semantics.description [hunk 3]
```diff
@@ -1,8 +1,8 @@
-// `check` semantics is not simply to return on error.
-// When `check` gets an error, it fails.
-// Enclosing block decide how to handle failure.
-// Most blocks pass failure up to enclosing block.
-// Function definition handles failure by returning the error.
-// `on fail` can catch the error.
-// `fail` statement is like `check` but always fails.
-// Differs from exceptions in that control flow is explicit
+// `check` semantics is simply not to return on an `error`.
```
Comment (MaryamZi) [inline examples/check-semantics/check_semantics.description:1]: Wasn't not in the correct place?

Same comment re: emphasizing error.

File: examples/check-semantics/check_semantics.description [hunk 4]
```diff
@@ -1,8 +1,8 @@
...
+// When `check` gets an `error`, it fails.
+// Enclosing block decides how to handle the failure.
```
Comment (MaryamZi) [inline examples/check-semantics/check_semantics.description:3]:
```suggestion
// The enclosing block decides how to handle the failure.
```

File: examples/rollback/rollback.bal [hunk 5]
```diff
@@ -8,13 +8,15 @@ public function main() returns error? {
...
+    [
+        {updateIndex: 0, stockMnt: 2000},
+        {updateIndex: 1, stockMnt: -1000},
+        {updateIndex: 2, stockMnt: 1500},
+        {updateIndex: 3, stockMnt: -1000},
+        {updateIndex: 4, stockMnt: -2000}
+    ];
+    // If an `error` is returned from the `transfer` function,
+    // the `error` is returned from the `main` and it exits.
     check transfer(updates);
```
Comment (MaryamZi) [inline examples/rollback/rollback.bal:20]:
```suggestion
    check transfer(updates);
    
```

File: examples/transaction-statement/transaction_statement.description (hunk 1 above)
Comment (MaryamZi) [reply examples/transaction-statement/transaction_statement.description:2]: There's no begin keyword, right? We shouldn't be emphasizing begin.

File: examples/check-semantics/check_semantics.description (hunk 3 above)
Comment (lochana-chathura) [reply examples/check-semantics/check_semantics.description]: didn't get what you mean by  `Wasn't in the correct place?`. Please check with the new changes.

Comment (MaryamZi) [reply examples/check-semantics/check_semantics.description]: Sorry, meant to say the previous version was correct.

Maybe
```
// `check` semantics is not simply to return on an `error` value.
```

Comment (lochana-chathura) [reply examples/check-semantics/check_semantics.description]: reading `not simply to return` felt a bit odd to me. Anyway, updated according to the above suggestion. thanks.

Comment (MaryamZi) [reply examples/check-semantics/check_semantics.description]: Or actually maybe
```
// `check` semantics is not to simply return on an `error` value.
```

@praneesha, wdyt? What we want to say is "`check` doesn't just mean returning when the expression evaluates to error".

Comment (lochana-chathura) [reply examples/check-semantics/check_semantics.description]: +1 on the new suggestion. Let's see if praneesha has anything to add.

File: examples/check-semantics/check_semantics.description [hunk 6]
```diff
@@ -1,8 +1,8 @@
-// `check` semantics is not simply to return on error.
-// When `check` gets an error, it fails.
-// Enclosing block decide how to handle failure.
-// Most blocks pass failure up to enclosing block.
-// Function definition handles failure by returning the error.
+// `check` semantics is not simply to return on an `error` value.
```
Comment (lochana-chathura) [inline examples/check-semantics/check_semantics.description:1]:
```suggestion
// `check` semantics is not to simply return on an `error` value.
```

**Comment by pubudu91** [comment]
Are we good to merge this?

**Review by MaryamZi — APPROVED** [review]

File: examples/check-semantics/check_semantics.description (hunk 3 above)
Comment (lochana-chathura) [reply examples/check-semantics/check_semantics.description]: Resolving the conversation as the new suggestion was added.

**Comment by lochana-chathura** [comment]
> Are we good to merge this?
yes!.

**Pull request merged**